app/wiring/AppLoader.scala

```diff
@@ -14,19 +14,22 @@ class AppLoader extends ApplicationLoader with StrictLogging {
   override def load(context: Context): Application = {
     new LogbackLoggerConfigurator().configure(context.environment)
 
-    val newContext = {
-      val identity = AppIdentity.whoAmI(defaultAppName = "admin-console")
-      val loadedConfig = ConfigurationLoader.load(identity) {
-        case AwsIdentity(app, stack, stage, _) => SSMConfigurationLocation(s"/$app/$stage")
-        case DevIdentity(app) =>
-          FileConfigurationLocation(new File(s"/etc/gu/support-admin-console.private.conf"))  //assume conf is available locally
-      }
+    val identity = AppIdentity.whoAmI(defaultAppName = "admin-console")
+    val loadedConfig = ConfigurationLoader.load(identity) {
+      case AwsIdentity(app, stack, stage, _) => SSMConfigurationLocation(s"/$app/$stage")
+      case DevIdentity(app) =>
+        FileConfigurationLocation(new File(s"/etc/gu/support-admin-console.private.conf"))  //assume conf is available locally
+    }
+
+    val newContext = context.copy(initialConfiguration = context.initialConfiguration ++ Configuration(loadedConfig))
 
-      context.copy(initialConfiguration = context.initialConfiguration ++ Configuration(loadedConfig))
+    val stage = identity match {
```

Comment:
since stage is used in so many places  it would be worth making into a value class to avoid mistakes that can't be found by the type checker
```scala
case class Stage(value: String) extends AnyVal
```